tests/cpp/test_index_select.cpp

```diff
@@ -857,4 +857,38 @@ TEST_F(NVFuserTest, IndexSelectVectorization3DCase1) {
   testValidate(&fusion, outputs, {t0, t1}, __LINE__, __FILE__);
 }
 
+TEST_F(NVFuserTest, IndexPutAccumulate) {
+  auto fusion_ptr = std::make_unique<Fusion>();
+  Fusion& fusion = *fusion_ptr.get();
+  FusionGuard fg(&fusion);
+
+  int64_t vocab = 1024;
+  int64_t hidden = 3584;
+  int64_t seq = 3000;
```

Comment:
We often encounter some unintended results when broadcast IDs are involved. Can you also add test cases with these parameters being 1?